src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx

```diff
@@ -66,7 +66,7 @@ export const ResourcesTable: React.FC<Props> = ({
     <Table {...props} variant={TableVariant.compact}>
       <Thead>
         <Tr>
-          <Th></Th>
+          <Th aria-hidden></Th>
```

Comment:
Now as I take a look, shouldn't the` {heads}` be inside of the `<Th>`. that or just change the that component to be self-closing

Reply:
Those empty Head tags are on purpose to have a column without title for the collapsible arrows